rts/lra-examples/cdi-embedded/pom.xml

```diff
@@ -76,6 +67,30 @@
         </dependency>
 
 
+        <!-- MP LRA CDI support defines annotations for controlling the lifecycle
+             of LRAs and for controlling participation in an LRA
+             -->
+        <dependency>
+            <groupId>org.eclipse.microprofile.lra</groupId>
+            <artifactId>microprofile-lra-api</artifactId>
+            <version>${version.microprofile.lra}</version>
+        </dependency>
+
+       <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I think the rest assured and junit5 dependencies are not needed as they are not used in the project (as far as I can understand).

Reply:
+1